Gemfile

```diff
@@ -7,6 +7,9 @@ gem 'berkshelf', '~> 3.0'
 gem 'omnibus', '5.5'
 gem 'omnibus-software', :github => 'opscode/omnibus-software' #, :branch => 'omnibus/3.2-stable'
 
+# Use open_uri_redirections gem to allow HTTPS -> HTTP redirections during package build.
```

Comment:
I meant adapting to the surrounding style, rather than gratuitously changing the upstream code. So, much as I like the trailing "."s, let's remove them.